### DISCUSSION TITLE
第6节课笔记(7班)

### DISCUSSION THREAD

**Discussion by JimmyMa99** [InternLM/Tutorial#462] [笔记]
第6节课笔记(7班)

**Comment by wuhaostudio** [comment]
https://blog.csdn.net/2301_77286822/article/details/135733585?spm=1001.2014.3001.5501

**Comment by SaaRaaS-1300** [comment]
第6节课，笔记+作业 Github：https://github.com/SaaRaaS-1300/InternLM_openNotebook/blob/main/Lesson-6/Lesson-6-Notebook.md

**Comment by createup2023** [comment]
https://www.yuque.com/creatup/ynrsde/hcdbngnh63zir6mw?singleDoc# 《第六次笔记：》
补充CSDN笔记：https://blog.csdn.net/weixin_48339493/article/details/135756552?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135756552%22%2C%22source%22%3A%22weixin_48339493%22%7D

**Comment by PommesPeter** [comment]
https://memo.sylin.host/docs/DeepLearning/%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86/%E8%AF%BE%E7%A8%8B%E5%AD%A6%E4%B9%A0/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0

**Comment by lifelinezhang** [comment]
https://zhuanlan.zhihu.com/p/679286518

**Comment by anneeeee** [comment]
https://mp.weixin.qq.com/s/X62NNHFWIeFlal-veM5RjQ

**Comment by hui1feng** [comment]
第七节课笔记：https://github.com/hui1feng/OpenCompass/blob/main/README.md

**Comment by abelezm** [comment]
第六课基础作业同笔记：https://blog.csdn.net/weixin_38043453/article/details/135820046

**Comment by leochiang** [comment]
第6课笔记
https://vvgyb242x83.feishu.cn/wiki/EfUBw8j9BibH5Qkw76Kcwes3nwh?from=from_copylink

**Comment by Simon-lyz** [comment]
## InternLM Lesson-6 Opencompass评测
https://g4w90egc30.feishu.cn/docx/ZOJZdv1duotLW4x0Nv8cWpyVnEb?from=from_copylink

**Comment by Rogue-Trader-zzy** [comment]
笔记：https://www.yuque.com/roguetrader/iu9odz/xt9u6ztcvonnbgfu?singleDoc# 《大模型评测教程》
